addons/project/static/src/components/project_many2one_field/project_many2one_field.xml

```diff
@@ -4,11 +4,9 @@
         <div class="d-flex align-items-center gap-1">
             <t t-set="isPrivateTask" t-value="props.readonly and !props.record.data.parent_id and !props.record.data.project_id"/>
             <Many2One t-if="!isPrivateTask" t-props="m2oProps"/>
-            <t t-if="isPrivateTask">
-                <span class="text-danger fst-italic text-muted">
-                    <i class="fa fa-lock"></i> Private
-                </span>
-            </t>
+            <span class="text-danger fst-italic text-muted"
+                t-out="this.props.record.fields[this.props.name].falsy_value_label"
+                t-if="isPrivateTask"/>
```

Comment:
Actually, I thought the ORM will return the falsy label when the field is empty. I am. surprised you have to do that. 🤔

Reply:
> Actually, I thought the ORM will return the falsy label when the field is empty. I am. surprised you have to do that. 🤔

I shared the details on the vidange server.thanks